peps/pep-0741.rst

```diff
@@ -735,6 +735,13 @@ behavior can change at each Python version: they are not "stable".
 Moreover, configuration options can be added, deprecated and removed
 following the usual :pep:`387` deprecation process.
 
+Soft Deprecate PyConfig API
+---------------------------
+
+The :pep:`587` ``PyConfig`` API is soft deprecated: it should no longer
+be used to write new code, but it remains safe to continue using it in
+existing code.
```

Comment:
As above, I don't think it makes sense to deprecate this API, since it's a genuinely better option for the CLI emulation use case. Instead I'd put a note here about the interaction with PEP 587 pre-initialization:

```suggestion
Interaction with the PyPreConfig and PyConfig APIs
--------------------------------------------------

The lower level :pep:`587` ``PyPreConfig`` and ``PyConfig`` APIs  remain available
and fully supported. As noted in the Abstract, they remain the preferred approach
for embedding use cases that are aiming to closely emulate the behaviour of the
full CPython CLI, rather than just making a Python runtime available as part of a
larger application.

The ``PyPreConfig`` APIs may be used in combination with the initialization
API in this PEP. In such cases, the read-only vs read/write restrictions for
preconfiguration settings apply to ``PyInitConfig_SetInt`` in addition to
``PyConfig_Set`` once the interpreter has been preconfigured
(specifically, only ``use_environment`` may be updated, attempting to
update any of the other preconfiguration variables will report an error)
```